build_tools/mako/configuration.py

```diff
@@ -127,6 +123,16 @@ def get_pixel4_default_target_list(skipped_target=None,
                  runtime_flags=[
                      "--task_topology_group_count=3",
                  ]),
+      TargetInfo(driver="vmvx",
+                 hal_target_backend="vmvx",
+                 taskset="f0",
+                 mako_tag="vmvx3t",
```

Comment:
Do you intend to add a benchmark on vmvx with 3 threads? This should match to the `value_key` in mako config files.

Reply:
yeah we'll want it threaded - will do